docs/source/visualisation/experiment_tracking.md

```diff
@@ -1,27 +1,26 @@
-# Set up experiment tracking
+# Experiment tracking in Kedro-Viz
 
 Experiment tracking is the process of saving all machine-learning related experiment information so you can compare different runs.
 
 [Kedro-Viz](https://github.com/kedro-org/kedro-viz) supports native experiment tracking from [version 4.1.1](https://github.com/kedro-org/kedro-viz/releases/tag/v4.1.1) onwards. When experiment tracking is enabled in your Kedro project, you will be able to access, edit and [compare your experiments](#access-run-data-and-compare-runs) and additionally [track how your metrics change over time](#view-your-metrics-timeline).
 
 ![](../meta/images/experiment-tracking_demo_small.gif)
 
-Enabling experiment tracking features on Kedro-Viz relies on:
+To enable the experiment tracking features of Kedro-Viz you need to:
 
-* [setting up a session store to capture experiment metadata](#set-up-the-session-store),
-* [experiment tracking datasets to let Kedro know what metrics should be tracked](#set-up-tracking-datasets),
-* [modifying your nodes and pipelines to output those metrics](#set-up-your-nodes-and-pipelines-to-log-metrics).
+- [Set up a session store to capture experiment metadata](#set-up-the-session-store),
+- [Set up experiment tracking datasets to list the metrics to track](#set-up-tracking-datasets),
+- [Modify your nodes and pipelines to output those metrics](#set-up-your-nodes-and-pipelines-to-log-metrics).
```

Comment:
```suggestion
- [set up a session store to capture experiment metadata](#set-up-the-session-store),
- [set up experiment tracking datasets to list the metrics to track](#set-up-tracking-datasets),
- [modify your nodes and pipelines to output those metrics](#set-up-your-nodes-and-pipelines-to-log-metrics).
```

Reply:
Sorry, @tynandebold This was my introduction -- I forgot the guidelines for bullets 🤦

Reply:
Now I'm confused. Should they be capitalized or lowercase?

Reply:
You know what, I'll roll it back to capitals, since other pages of the docs do have them capitalised. I've no idea when/if that's a change, as I may be thinking of another style guide, but that doesn't explain why the bullets in the original copy were lower case here. Never mind. I'm now happy for you to merge this when you are ready.

Reply:
Copy that!